### DISCUSSION TITLE
为什么依赖里面有lodash和lodash-es

### DISCUSSION THREAD

**Discussion by Ggmid** [vueComponent/ant-design-vue#7139] [Q&A]
出于什么考虑，为什么依赖里面有lodash和lodash-es

**Comment by webvs2** [comment]
![image](https://github.com/vueComponent/ant-design-vue/assets/30883395/d946000b-e8cb-4610-9868-bc048df14a2c)
没有发现lodash-es，是在哪里查出来的呢

**Comment by Ggmid** [comment]
<img width="822" alt="image" src="https://github.com/vueComponent/ant-design-vue/assets/22318151/7518d0a3-296f-4425-b307-42a26f052c24">

**Reply by Ggmid** [reply]
不是在你自己的项目里面，是在ant-design-vue的源码里面

**Comment by ryne6** [comment]
可以看看这两个issue
<img width="773" alt="image" src="https://github.com/vueComponent/ant-design-vue/assets/91053541/5026e1cf-94ec-46ff-8c67-58fc9f143321">

**Reply by Ggmid** [reply]
👌，那么我想问一下，如果我不想要打包出来的文件里面有lodash的存在，是否只能按需加载，并且不使用一些用到lodash的组件吗

**Reply by extclp** [reply]
lodash是函数库, 没必要在意
